src/query/settings/src/lib.rs

```diff
@@ -594,12 +594,10 @@ impl Settings {
         let key = "sql_dialect";
         self.check_and_get_setting_value(key)
             .and_then(|v| v.user_setting.value.as_string())
-            .map(|v| {
-                if v == "MySQL" {
-                    Dialect::MySQL
-                } else {
-                    Dialect::PostgreSQL
-                }
+            .map(|v| match &*v {
```

Comment:
How about converting to lowercase and comparing with `mysql`, `hive` instead?

Reply:
done